uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyMerged.java

```diff
@@ -39,10 +40,11 @@
 
 public class ColumnRenderingStrategyMerged {
```

Comment:
Not sure where to put this comment, looking on classes `ColumnRenderingStrategyMerged` and `ColumnRenderingStrategyFlattened` it seems to me that there is a lot of repeated code, I think it is not corollary of this PR, it makes more formating changes to these classes it seems to me, but I am thinking, if it shouldn'e be worth to extract at least pieces where a lot of constants is used and thus update of them could be painful in future especially:
```java
bodyGrid.M(x, y + 0.5)
                        .L(x + columnWidth, y + 0.5);
```

and
```java
final int columnIndex = model.getColumns().indexOf(column);
            if (columnIndex < model.getColumnCount() - 1) {
                bodyGrid.M(x + columnWidth + 0.5, 0)
                        .L(x + columnWidth + 0.5, columnHeight);
            }
```

Reply:
@jomarko Repeated code across the two classes; or repeated inside the class, or both!? ;-)

I do agree these two classes (especially ```ColumnRenderingStrategyMerged```) are not pleasant to look at and am open to suggestion.. interestingly IDEA does not report any duplicated code (but it sure _looks_ duplicated)!